server/lib/channel.dart

```diff
@@ -1,38 +1,44 @@
 import 'server.dart';
+import 'service/google_api.dart';
 
-/// This type initializes an application.
-///
-/// Override methods in this class to set up routes and initialize services like
-/// database connections. See http://aqueduct.io/docs/http/channel/.
+/// A subclass of [ApplicationChannel] that is created for each isolate.
 class ServerChannel extends ApplicationChannel {
-  /// Initialize services in this method.
-  ///
-  /// Implement this method to initialize services, read values from [options]
-  /// and any other initialization required before constructing [entryPoint].
-  ///
-  /// This method is invoked prior to [entryPoint] being accessed.
+  /// The [GoogleApi] instance that provides authenticated clients to access
+  /// Google APIs.
+  GoogleApi googleApi;
+
+  ServerConfiguration configuration;
+
+  /// Initializes logger, configuration values, and services to be injected.
   @override
   Future prepare() async {
-    logger.onRecord.listen(
-        (rec) => print("$rec ${rec.error ?? ""} ${rec.stackTrace ?? ""}"));
+    // Record stack traces if level is severe or higher.
+    recordStackTraceAtLevel = Level.SEVERE;
+    logger.onRecord.listen((final record) {
+      print(record);
```

Comment:
would print record already include record.error and record.stackTrace?

Reply:
No, by itself, it would just print the [record message](https://pub.dev/documentation/logging/latest/logging/LogRecord/toString.html).